### PR TITLE
Update AUTHORS

### PR DESCRIPTION
**Pull request by justincormack**
Signed-off-by: Justin Cormack <justin.cormack@docker.com>

owlet

### CODE REVIEW

**Comment by justincormack** [comment]
There were two Dave/David Scotts - which do you prefer @djs55

**Comment by justincormack** [comment]
The script is not very good at normalising name differences, only email differences...

File: AUTHORS [hunk 1]
```diff
@@ -36,7 +36,9 @@ Mindy Preston <mindy.preston@docker.com>
 Natanael Copa <natanael.copa@docker.com>
 Nathan Dautenhahn <ndd@seas.upenn.edu>
 Nathan LeClaire <nathan.leclaire@docker.com>
+Nico Di Rocco <dirocco.nico@gmail.com>
 Phil Estes <estesp@linux.vnet.ibm.com>
+radu-matei <matei.radu94@gmail.com>
```
Comment (thaJeztah) [inline AUTHORS]: If you want to have this nicer looking; name looks to be `Radu Matei`; https://github.com/radu-matei

Comment (justincormack) [inline AUTHORS]: updated

**Comment by djs55** [comment]
Could you stick me down as `David Scott <dave.scott@docker.com>`?  It's what I've been using elsewhere.

**Comment by justincormack** [comment]
@djs55 amended (you could change your github profile to match...)

**Comment by djs55** [comment]
@justincormack ha, good point. I've done that too. Marginally more consistent now

File: AUTHORS (hunk 1 above)
Comment (thaJeztah) [inline AUTHORS]: Oh, that needed an entry in `.mailmap` so that it automatically maps the e-mail to the preferred name. Let me do a follow-up

**Pull request merged**